step.yml

```
@@ -92,10 +92,10 @@ inputs:
       description: |
         Release name used only if ZIP is uploaded to TryoutApps.
       is_required: false
-  - ruby_version:
+  - gemfile_path:
```

Is it possible to create gemfile upfront and store it inside the step folder, then use it as default value in here?
something like this:
```suggestion
  - gemfile_path: $BITRISE_STEP_PATH/.helpers/Gemfile
```

something similar to the shaman_init, but we won't read it from the project root folder

---

> store it inside the step folder

Hmm, can you provide more details on the idea, please? Is the step path where the step is executed or? 
Shaman.yml, we keep locally in the project so we can easily define the default path.  
From my understanding, since the step is being executed in the project root, we could either create the file dynamically or keep it in the project locally. Am I missing something?

---

My proposal can't be used if the step gets all values from the project root folder. So let's keep it as proposed (dynamically create Gemfile).